examples/no-framework/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+use GraphQL\GraphQL;
+use GraphQL\Type\Schema;
+use TheCodingMachine\GraphQLite\SchemaFactory;
+use TheCodingMachine\GraphQLite\Context\Context;
+
+use Symfony\Component\Cache\Simple\FilesystemCache;
+use Mouf\Picotainer\Picotainer;
+use GraphQL\Utils\SchemaPrinter;
+use App\Controllers\MyController;
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+// $cache is any PSR-16 compatible cache.
+$cache = new FilesystemCache();
+
+// $container is any PSR-11 compatible container which has
+// been populated with your controller classes.
+$container = new Picotainer([
+        MyController::class => function() {
+                return new MyController();
+        },
+]);
+
+$factory = new SchemaFactory($cache, $container);
+$factory->addControllerNamespace('App\\Controllers\\')
```

Comment:
So, I don't think the trailing slash is needed on these namespaces.  I'm unsure if that's causing the issue or not.

Reply:
I've pushed an update to docs on this already.